src/utils/mod.rs

```diff
@@ -1,19 +1,15 @@
 use std::path::Path;
-use std::time::Duration;
 use std::{fs, io};
 
 use ethers::addressbook::Address;
 use ethers::types::U256;
```

Comment:
We should update to `alloy` at some point since ethers is deprecated

Reply:
yes we need to do that but we need to update zaun first, hence another PR for that refactor